Bricks/ForwardModeAD/0.1.0.toml

```diff
@@ -0,0 +1,12 @@
+
+[brick]
+authors = "Luca Ferranti"
+chplVersion = "1.28.0"
+license = "MIT"
+name = "ForwardModeAD"
+source = "git@github.com:lucaferranti/ForwardModeAD.git"
```

Comment:
The error message

```
 git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

maybe this should be specified as https link?

Reply:
It might be it , because also the example in the readme uses https.

I think this file was automatically generated though, maybe this is something should be fixed in `mason publish`?